tools/squidclient/Transport.cc

```diff
@@ -249,8 +249,8 @@ Transport::Write(const void *buf, size_t len)
 #endif
     } else {
 
-#if _SQUID_WINDOWS_
-        return send(conn, buf, len, 0);
+#if _SQUID_WINDOWS_ || _SQUID_MINGW_
```

Comment:
This condition should not be added because its addition changes nothing in the official code: In the official code, `_SQUID_MINGW_` cannot be true if `_SQUID_WINDOWS_` is false. The situation will change after PR 1341 lands. This PR can be merged before that if these `_SQUID_MINGW_` are removed, but since we will probably want to add these conditions after PR 1341 lands, we probably should wait for that PR to land first (to avoid changing this code twice). Your call though.

Reply:
PR #1341 has landed. Making this request obsolete.